Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,262 @@
+                        "PanOSOutboundInterfaceDetailsType",
+                        PanOSOutboundInterfaceDetailsType,
+                        "PanOSParentSessionID",
+                        PanOSParentSessionID,
+                        "PanOSContainerName",
+                        PanOSContainerName,
+                        "PanOSContainerNameSpace",
+                        PanOSContainerNameSpace,
+                        "PanOSHTTPRefererFQDN",
+                        PanOSHTTPRefererFQDN,
+                        "PanOSHTTPRefererPort",
+                        PanOSHTTPRefererPort,
+                        "PanOSHTTPRefererProtocol",
+                        PanOSHTTPRefererProtocol,
+                        "PanOSHTTPRefererURLPath",
+                        PanOSHTTPRefererURLPath,
+                        "PanOSRuleUUID",
+                        PanOSRuleUUID,
+                        "PanOSURLCategoryList",
+                        PanOSURLCategoryList,
+                        "PanOSURLDomain",
+                        PanOSURLDomain,
+                        "PanOSURLCounter",
+                        PanOSURLCounter,
+                        "PanOSUsers",
+                        PanOSUsers,
+                        "PanOSVendorSeverity",
+                        PanOSVendorSeverity,
+                        "PanOSX-Forwarded-For",
+                        ["PanOSX-Forwarded-For"],
+                        "PanOSX-Forwarded-ForIP",
+                        ["PanOSX-Forwarded-ForIP"],
+                        "PanOSLogSource",
+                        PanOSLogSource
+                    ),
+          HttpContentType = RequestContext
+      | project-rename
+          DvcIpAddr = Computer,
+          EventUid = _ItemId,
+          DstDvcId = PanOSDestinationUUID,
+          DstGeoCountry = PanOSDestinationLocation,
+          DstMacAddr = PanOSDestinationDeviceMac,
+          DstNatIpAddr = DestinationTranslatedAddress,
+          DstNatPortNumber = DestinationTranslatedPort,
+          DstPortNumber = DestinationPort,
+          DstUsername = DestinationUserName,
+          DstZone = DeviceCustomString5,
+          DvcId = DeviceExternalID,
+          DvcOriginalAction = DeviceAction,
+          EventOriginalSeverity = LogSeverity,
+          EventOriginalType = DeviceEventClassID,
+          EventOriginalUid = ExtID,
+          EventProductVersion = DeviceVersion,
+          HttpContentFormat = RequestContext,
+          HttpReferrer = PanOSReferer,
+          RuleName = DeviceCustomString1,
+          SrcDvcId = PanOSSourceUUID,
+          SrcMacAddr = PanOSSourceDeviceMac,
+          SrcNatIpAddr = SourceTranslatedAddress,
+          SrcNatPortNumber = SourceTranslatedPort,
+          SrcPortNumber = SourcePort,
+          SrcUsername = SourceUserName,
+          SrcZone = DeviceCustomString4,
+          Url = RequestURL,
+          UrlCategory = DeviceCustomString2,
```

Comment:
as per official doc we have this. Does DeviceCustomString2 also has same value?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/e84d9765-929a-491e-b092-710d816a904a)

Reply:
validated that we are getting UrlCategory in DeviceCustomString2. Resolving this comment